src/main/webapp/WEB-INF/conf/samply.share.broker.conf

```diff
@@ -8,8 +8,12 @@ mdr.url=https://mdr.ccp-it.dktk.dkfz.de/v3/api/mdr/
 
 # Icinga configuration
 icinga.host=https://monitor.vmitro.de:8443
-icinga.path=/v1/actions/process-check-result
+icinga.path=/v1/actions/process-check-report
 icinga.username=dktk-searchbroker-java
 icinga.password=EQrfn3WW226ChnXbV4JYhgIPhQSc
 # (A) for dev system (B) for production
-icinga.site_suffix=GBA (A)
+icinga.site_suffix=BBMRI (A)
+icinga.project=-bbmri
+
+abide.username=test123
+abide.password=test123
```

Comment:
Do we really need to have a fixed password here? Please explain the reason for the separate account.

Reply:
Because Icinga is an diffrent api and software. And it is better to use different credentials in different projects?

Reply:
We could call the credentials structured-query-execution.username and .password. But why are the credentials not managed like the other credentials. Why do we have them fixed in the config files and not editable in the UI? Or are only credentials in the UI that are used by the share-client in order to access other API's?